Core/Systems/CombatMountSystem/CombatMountPlayer.cs

```diff
@@ -6,6 +6,11 @@ internal class CombatMountPlayer : ModPlayer
 	{
 		public CombatMount activeMount;
 
+		/// <summary>
+		/// Multiplier for combat mount cooldowns
```

Comment:
could this comment clarify if up or down is faster with an example? Just for the sanity of people touching this in the future especially after our fiasco with pickaxe speed